nifi-nar-bundles/nifi-standard-services/nifi-proxy-configuration-bundle/nifi-proxy-configuration/src/main/java/org/apache/nifi/proxy/StandardProxyConfigurationService.java

```diff
@@ -46,31 +46,41 @@ public class StandardProxyConfigurationService extends AbstractControllerService
             .required(true)
             .build();
 
-    static final PropertyDescriptor PROXY_SERVER_HOST = new PropertyDescriptor.Builder()
+    public static final PropertyDescriptor SOCKS_VERSION = new PropertyDescriptor.Builder()
+        .name("socks-version")
+        .displayName("Socks version")
+        .description("Socks version.")
```

Comment:
Recommend expanding the description:
```suggestion
        .description("SOCKS Protocol Version")
```